### ISSUE TITLE
Migrate to Dokka v2

### ISSUE THREAD

**Issue by BraisGabin** [detekt/sarif4k#196]
Right now we are getting this warning message:

```
warning: Dokka Gradle plugin V1 is deprecated

    Dokka Gradle plugin V1 is deprecated, and will be removed in Dokka version 2.1.0
    Please migrate to Dokka Gradle plugin V2. This will require updating your project.
    To learn about migrating read the migration guide https://kotl.in/dokka-gradle-migration

    To start migrating to Dokka Gradle plugin V2 add
        org.jetbrains.dokka.experimental.gradle.pluginMode=V2EnabledWithHelpers
    into your project's `gradle.properties` file.

    We would appreciate your feedback!
     - Please report any feedback or problems https://kotl.in/dokka-issues
     - Chat with the community visit #dokka in https://kotlinlang.slack.com/ (To sign up visit https://kotl.in/slack)
```

We need to migrate to v2

**Comment by atulgpt** [comment]
Currently Dokka V2 is experimental as they have mentioned `The Dokka Gradle plugin v2 is an [Experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained) feature.`

**Comment by TWiStErRob** [comment]
Looks like migration is almost a no-op :) Done in #203
